include/variables.h

```diff
@@ -192,7 +193,31 @@ extern s32 gPauseMenuTargetPosY;
 extern s32 gPauseMenuCursorTargetOpacity;
 extern u8 gPauseMenuTextScrollInterpEasingLUT[16];
 extern u8 gPauseMenuPageScrollInterpEasingLUT[16];
-
+extern s32 D_80270700[1];
+extern f32 D_80270704;
+extern f32 D_80270708;
+extern s32 mapMarioXPos; //mario xPos on map tab
+extern s32 mapMarioYPos; //mario xPos on map tab
+extern f32 mapCursorXPos; //cursor X pos float map tab
+extern f32 mapCursorYPos; //cursor Y pos float map tab
+extern s32 mapCursorCurrentOption;
+extern s32 mapCursorCurrentOptionCopy;
```

Comment:
Are globals/statics still using the g/s naming method? If so, these should be renamed.